pyomo/solvers/plugins/solvers/GUROBI_RUN.py

```diff
@@ -167,6 +167,11 @@ def gurobi_run(model_file, warmstart_file, soln_file, mipgap, options, suffixes)
         message = 'Optimization terminated because the time expended exceeded the value specified in the TimeLimit parameter.'
         term_cond = 'maxTimeLimit'
         solution_status = 'stoppedByLimit'
+    elif hasattr(GRB, "WORK_LIMIT") and (solver_status == GRB.WORK_LIMIT):
+        status = 'aborted'
+        message = 'Optimization terminated because the work expended exceeded the value specified in the WorkLimit parameter.'
+        term_cond = 'maxTimeLimit'
```

Comment:
Would `'maxIterations'` or `'maxEvaluations'` be more appropriate than `'maxTimeLimit'`?

Reply:
I thought about this but in the docs it is an alternative to a time limit. 🤷‍♂️

Reply:
Good enough for me.